### ISSUE TITLE
install_foo functions install every option

### ISSUE THREAD

**Issue by prometheanfire** [sadko4u/lsp-plugins#31]
`build_foo` works, `install_foo` will try to install (and compile if not compiled) all modules.  (Ignore the missing JACK stuff below, I'm trying to build without JACK support)  removing the all parameter seems to help.

```
make -j5 -l16 install_lv2 
Package jack was not found in the pkg-config search path.
Perhaps you should add the directory containing `jack.pc'
to the PKG_CONFIG_PATH environment variable
No package 'jack' found
Package jack was not found in the pkg-config search path.
Perhaps you should add the directory containing `jack.pc'
to the PKG_CONFIG_PATH environment variable
No package 'jack' found
-------------------------------------------------------------------------------
Building binaries
  target architecture : x86_64
  target platform     : Linux
  target system       : Linux
  modules             : ladspa lv2 vst jack profile src doc
-------------------------------------------------------------------------------
```

**Comment by prometheanfire** [comment]
`sed -i '/install_.*: all/s/ all//g' Makefile`

**Comment by sadko4u** [comment]
Hi there!
To build without JACK stuff, you need to try the following sequence of commands:
```
make clean
make BULD_MODULES='ladspa lv2 vst doc'
make install
```

**Comment by prometheanfire** [comment]
I tried that and it didn't seem to work.

```
make -j5 -l16 clean 
Package jack was not found in the pkg-config search path.
Perhaps you should add the directory containing `jack.pc'
to the PKG_CONFIG_PATH environment variable
No package 'jack' found
Package jack was not found in the pkg-config search path.
Perhaps you should add the directory containing `jack.pc'
to the PKG_CONFIG_PATH environment variable
No package 'jack' found
Clean OK
make -j5 -l16 'BUILD_MODULES= ladspa lv2' 
Package jack was not found in the pkg-config search path.
Perhaps you should add the directory containing `jack.pc'
to the PKG_CONFIG_PATH environment variable
No package 'jack' found
Package jack was not found in the pkg-config search path.
Perhaps you should add the directory containing `jack.pc'
to the PKG_CONFIG_PATH environment variable
No package 'jack' found
-------------------------------------------------------------------------------
Building binaries
  target architecture : x86_64
  target platform     : Linux
  target system       : Linux
  modules             : ladspa lv2
-------------------------------------------------------------------------------
make -j5 -l16 PREFIX=/usr DESTDIR=/var/tmp/portage/media-libs/lsp-plugins-1.1.5/image install 
Package jack was not found in the pkg-config search path.
Perhaps you should add the directory containing `jack.pc'
to the PKG_CONFIG_PATH environment variable
No package 'jack' found
Package jack was not found in the pkg-config search path.
Perhaps you should add the directory containing `jack.pc'
to the PKG_CONFIG_PATH environment variable
No package 'jack' found
-------------------------------------------------------------------------------
Building binaries
  target architecture : x86_64
  target platform     : Linux
  target system       : Linux
  modules             : ladspa lv2 vst jack profile src doc
-------------------------------------------------------------------------------
```

**Comment by sadko4u** [comment]
The command:
```
make -j5 -l16 'BUILD_MODULES= ladspa lv2' 
```
Is invalid, should be:
```
make -j5 -l16 BUILD_MODULES='ladspa lv2' 
```

**Comment by sadko4u** [comment]
Anyway, I've reproduced the case. Will look more detailed

**Comment by sadko4u** [comment]
The problem has been fixed in this commit: 43f94217555f08f2a30a8505b26ec1dca30319a1
Now works properly, partial builds and installations are OK.
Please confirm.

**Comment by prometheanfire** [comment]
yep, https://github.com/sadko4u/lsp-plugins/commit/43f94217555f08f2a30a8505b26ec1dca30319a1 works

**Comment by prometheanfire** [comment]
also don't need to do a make clean (packaging it for gentoo, we always start the build 'clean')

**Comment by sadko4u** [comment]
Merged changes into **devel** branch: 9a8c5f0cb30fa6d56f912f50a3ffed5490485226

**Comment by prometheanfire** [comment]
ok, one last question, I'm packaging what would normally be a git master version, would devel be the branch for that?  (something like https://github.com/gentoo/gentoo/blob/master/media-sound/pulseeffects/pulseeffects-9999.ebuild )  Also, should this be closed now that it's in devel?

**Comment by sadko4u** [comment]
You can use the **devel** branch for building. Actually, it is a pre-master branch. All changes from **devel** will be merged into **master** when the release will be shipped.
I'll close all **PENDING FOR RELEASE** issues after the release.

**Comment by prometheanfire** [comment]
cool, thanks

**Comment by sadko4u** [comment]
All changes are available in the 1.1.7 release that was just published.
